projects/fizzbuzz/java/FizzBuzz.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This is a function to iterate 100 times, if the iteration
+ * is divisible by 3 or 5, print Fizz, or Buzz respectivly, 
+ * else print the number of the iteration starting at 1, ending at 100.
+ */
+package fizzbuzz;
+
+/**
+ *
+ * @author visionaryJack
+ */
+public class FizzBuzz {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        String fizz = "Fizz"; //String for 3

```

Comment:
This could be included directly into line: 29 and as part of line: 25 to reduce overall lines of code. If file size is an issue for your particular application of the program.